src/test/java/com/meilisearch/integration/ClientTest.java

```diff
@@ -201,6 +202,37 @@ public void testDeleteIndex() throws Exception {
         assertThrows(MeilisearchApiException.class, () -> client.getIndex(indexUid));
     }
 
+    /** Test swapIndexes */
+    @Test
+    public void testSwapIndexes() throws Exception {
```

Comment:
I appreciate the effort to test it, but it could be more straightforward. Just the assertion to make sure you could request using the correct indexes and retrieve the type of the task `indexSwap` would have been ok.

Reply:
I knew you were going to say that when I wrote the test but I couldn't help myself 🤭